bluebrain/deployment/environments/applications_libraries.yaml

```diff
@@ -36,7 +36,7 @@ spack:
           all: '{name}/{version}'
   specs:
     - boost+atomic+chrono+date_time+filesystem+json+locale+log+math+program_options+python+random+regex+serialization+shared+signals+stacktrace+system+test+timer+type_erasure
-    - caliper+cuda cuda_arch=70
+    - caliper+cuda@2.8.0 cuda_arch=70
```

Comment:
@matz-e : This is not sufficient to enforce 2.8.0 as preferred. What is the current way to set the preferred version? 
* I guess we no longer have packages.yaml
* Should I update the recipe in the builtin repo directly? Or, just create one in repo-patches?